src/test/fixtures/config/generator.tsx

```diff
@@ -0,0 +1,214 @@
+      case "TRANSFERABLE_RECORD":
+        const transferableRecordCredential = credential as Credential;
+
+        const transferableIssuer = {
+          id: "https://example.com",
+          name: "DEMO TOKEN REGISTRY",
+          type: "OpenAttestationIssuer",
+        };
+
+        const transferableProof = {
+          type: "OpenAttestationProofMethod",
+          method: "TOKEN_REGISTRY",
+          value: transferableRecordCredential.tokenRegistry,
+        };
+
+        const transferableIdentityProof = {
+          type: "DNS-TXT",
+          identifier: transferableRecordCredential.identityProof,
+        };
+
+        return {
+          ...form,
+          defaults: {
+            ...form.defaults,
+            issuer: transferableIssuer,
+            openAttestationMetadata: {
+              ...form.defaults.openAttestationMetadata,
+              proof: transferableProof,
+              identityProof: transferableIdentityProof,
+            },
+          },
+        };
+
+      case "VERIFIABLE_DOCUMENT":
+        let verifiableIssuer;
+        let verifiableProof;
+        let verifiableIdentityProof;
+
+        if (generateDid) {
+          const didCredential = credential as DidCredential;
+          verifiableIssuer = {
+            id: "https://example.com",
+            name: "DEMO DID",
+            type: "OpenAttestationIssuer",
+          };
+          verifiableProof = {
+            type: "OpenAttestationProofMethod",
+            method: "DID",
+            value: didCredential.didAddress,
+            revocation: didCredential.revocation,
+          };
+          verifiableIdentityProof = {
+            type: "DNS-DID",
+            identifier: didCredential.identityProof,
+          };
+        } else {
+          const verifiableDocumentCredential = credential as Credential;
+
+          verifiableIssuer = {
+            id: "https://example.com",
+            name: "DEMO DOCUMENT STORE",
+            type: "OpenAttestationIssuer",
+          };
+          verifiableProof = {
+            type: "OpenAttestationProofMethod",
+            method: "DOCUMENT_STORE",
+            value: verifiableDocumentCredential.documentStore,
+          };
+          verifiableIdentityProof = {
+            type: "DNS-TXT",
+            identifier: verifiableDocumentCredential.identityProof,
+          };
+        }
+
+        return {
+          ...form,
+          defaults: {
+            ...form.defaults,
+            issuer: verifiableIssuer,
+            openAttestationMetadata: {
+              ...form.defaults.openAttestationMetadata,
+              proof: verifiableProof,
+              identityProof: verifiableIdentityProof,
+            },
+          },
+        };
+      default:
+        throw Error("no form type found");
+    }
+  });
+}
+
+function updateConfigFile(configFile: any, directory: string, validationBypass = false) {
+  if (!validationBypass) {
+    assertConfigFile(configFile);
+  }
```

Comment:
we can tap on this by running  `npm run build:config` on ci/cd, so if validate fails, at least pr will be blocked wdyt?

Reply:
mmm sound okay just the build will be slower.